app/uk/gov/hmrc/agentepayeregistration/services/AdminService.scala

```diff
@@ -19,7 +19,6 @@ package uk.gov.hmrc.agentepayeregistration.services
 import javax.inject.Inject
 import play.api.Logger
 import play.modules.reactivemongo.ReactiveMongoComponent
-import reactivemongo.api.commands.UpdateWriteResult
 import uk.gov.hmrc.agentepayeregistration.repository.AgentEpayeRegistrationRepository
 
```

Comment:
The deleteStaleDocuments job in this file could be deleted.

Reply:
Actually, this whole thing could be deleted?

Reply:
Probably could - this was a 2.6 uplift story though